apps/backoffice-v2/src/pages/Root/Root.error.tsx

```diff
@@ -1,11 +1,28 @@
 import { Button } from '../../common/components/atoms/Button/Button';
-import { Link } from 'react-router-dom';
+import { Link, Navigate, useRouteError } from 'react-router-dom';
 import { FunctionComponent } from 'react';
 import { Providers } from '../../common/components/templates/Providers/Providers';
 import { Toaster } from 'react-hot-toast';
 import { ErrorAlert } from '../../common/components/atoms/ErrorAlert/ErrorAlert';
+import { useAuthenticatedLayoutLogic } from '../../domains/auth/components/AuthenticatedLayout/hooks/useAuthenticatedLayoutLogic/useAuthenticatedLayoutLogic';
+import { isErrorWithMessage } from '@ballerine/common';
 
 export const RootError: FunctionComponent = () => {
+  const error = useRouteError();
+  const { redirectUnauthenticatedTo, location } = useAuthenticatedLayoutLogic();
+
+  if (isErrorWithMessage(error) && error.message === 'Unauthorized (401)') {
```

Comment:
it would be better to check status code instead of message

Reply:
Good chance the error does not have the code. Will ensure first thing tomorrow. 👍🏼